javaslang/generator/Generator.scala

```diff
@@ -58,6 +58,67 @@ def generateMainClasses(): Unit = {
       val TryType = im.getType("javaslang.control.Try")
       val ValidationType = im.getType("javaslang.control.Validation")
       val CharSeqType = im.getType("javaslang.collection.CharSeq")
+      val ArrayType = im.getType("javaslang.collection.Array")
+      val VectorType = im.getType("javaslang.collection.Vector")
+      val ListType = im.getType("javaslang.collection.List")
+      val StreamType = im.getType("javaslang.collection.Stream")
+      val QueueType = im.getType("javaslang.collection.Queue")
+      val LinkedHashSetType = im.getType("javaslang.collection.LinkedHashSet")
+      val HashSetType = im.getType("javaslang.collection.HashSet")
+      val JavaStreamType = im.getType("java.util.stream.Stream")
+
+      def genTraversableAliases(traversableType: String, name: String): String = {
+        xs"""
+          /$javadoc
+           * Alias for {@link $traversableType#of(Object)}
+           *
+           * @param <T>     Component type of element.
+           * @param element An element.
+           * @return A new $traversableType instance containing the given element
+           */
+          public static <T> $traversableType<T> $name(T element) {
+              return $traversableType.of(element);
+          }
+
+          /$javadoc
+           * Alias for {@link $traversableType#of(Object...)}
+           *
+           * @param <T>      Component type of elements.
+           * @param elements Zero of more elements.
+           * @return A new $traversableType instance containing the given elements
+           * @throws NullPointerException if {@code elements} is null
+           */
+          @SuppressWarnings("varargs")
+          @SafeVarargs
+          public static <T> $traversableType<T> $name(T... elements) {
+              return $traversableType.of(elements);
+          }
+
+          /$javadoc
+           * Alias for {@link $traversableType#ofAll(Iterable)}
+           *
+           * @param <T>      Component type of elements.
+           * @param elements Zero of more elements.
+           * @return A new $traversableType instance containing the given elements
```

Comment:
`{@code Something}` in JavaDoc will generate text with style `code` - only text without any link.
In IDEA no links too.
Now I set `{@link ClassName}` in all `return` JavaDoc parts